tests/mgr/test_cephmgr_validate_mgr_status.py

```diff
@@ -0,0 +1,66 @@
+from ceph.waiter import WaitUntil
+from cli.cephadm.cephadm import CephAdm
+from cli.utilities.utils import get_service_id, reboot_node, set_service_state
+
+
+class CephMgrError(Exception):
```

Comment:
This is very generic exception. Change this to `CephMgrServiceFailError`. Create more exception for other operations.